package.json

```diff
@@ -68,14 +68,14 @@
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
     "copy:locales": "node pontoon-to-webext",
-    "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && svgo src/icons -o dist/icons && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
```

Comment:
Am I correct that this _wonʼt_ copy the `src/icons/color_bell_icon.png`?  Should we change that to `src/icons/*.png` instead?